laskyawm.css

```diff
@@ -31,8 +31,8 @@ body:before {
 	position: absolute !important;
 	top: 25px;
 	left: 100px;
-	box-shadow: 0px 0px 20px rgba(0, 0, 0, .3);
 	background-color: rgba(255, 255, 255, .2);
+	border: 1px solid transparent;
```

Comment:
Did you see the slight accented border:
![untitled](https://cloud.githubusercontent.com/assets/1479244/18059443/fb90a01e-6de6-11e6-8b60-23fd758cad55.png)